doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1307 @@
+
+      * :c:func:`lwm2m_init_location`
+      * :c:func:`lwm2m_init_device`
+      * :c:func:`lwm2m_init_cellular_connectivity_object`
+      * :c:func:`lwm2m_init_connmon`
+
+    * :c:func:`lwm2m_init_firmware` is deprecated in favour of :c:func:`lwm2m_init_firmware_cb` that allows application to set a callback to receive FOTA events.
+
+* :ref:`lib_nrf_cloud_pgps` library:
+
+  * Fixed a NULL pointer issue that could occur when there were some valid predictions in flash but not the one required at the current time.
+
+* :ref:`lib_download_client` library:
+
+  * Removed the deprecated ``download_client_connect`` function.
+
+* :ref:`lib_fota_download` library:
+
+  * Added:
+
+    * The function :c:func:`fota_download_b1_file_parse` to parse a bootloader update file path.
+    * Experimental support for performing FOTA updates using an external download client with the Kconfig option :kconfig:option:`CONFIG_FOTA_DOWNLOAD_EXTERNAL_DL` and functions :c:func:`fota_download_external_start` and :c:func:`fota_download_external_evt_handle`.
+
+Security libraries
+------------------
+
+* :ref:`trusted_storage_readme` library:
+
+  * Added the Kconfig option :kconfig:option:`CONFIG_TRUSTED_STORAGE_STORAGE_BACKEND_CUSTOM` that enables use of custom storage backend.
+
+SUIT DFU
+--------
+
+* Added:
+
+  * Experimental support for :ref:`SUIT DFU <ug_nrf54h20_suit_dfu>` for nRF54H20.
+    SUIT is now the only way to boot local domains of the nRF54H20 SoC.
+  * Experimental support for DFU from external flash for nRF54H20.
+  * Experimental support for recovery image for application and radio cores for nRF54H20.
+
+Other libraries
+---------------
+
+* Added:
+
+  * The :ref:`dult_readme` library with the experimental maturity level.
+  * The :ref:`lib_uart_async_adapter` library.
+
+* :ref:`app_event_manager` library:
+
+  * Added the :kconfig:option:`CONFIG_APP_EVENT_MANAGER_REBOOT_ON_EVENT_ALLOC_FAIL` Kconfig option.
+    The option allows to select between system reboot or kernel panic on event allocation failure for default event allocator.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added:
+
+  * The :file:`thingy91x_dfu.py` script in the :file:`scripts/west_commands` folder.
+    The script adds the west commands ``west thingy91x-dfu`` and ``west thingy91x-reset`` for convenient use of the serial recovery functionality.
+  * Support for format version ``1`` of the :file:`dfu_application.zip` file to :ref:`nrf_desktop_config_channel_script` Python script.
+    From |NCS| v2.7.0 onwards, the script supports both format version ``0`` and ``1`` of the zip archive.
+    Older versions of the script do not support format version ``1`` of the zip archive.
+
```

Comment:
@b-gent please add a new section for Nordic Secure Immutable Bootloader (NSIB)
```
NSIB
===
```

Then a following text.

```
* Updated:

  * Improved a key revocation handling process to remove security weakness found in the previous design. It is recommended to switch to the improved revocation handling in the newly manufactured devices.
```

(@nvlsianpu , @oyvindronningstad  )

Reply:
Adding under Samples -> Other as this is where it's located in docs
https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bootloader/README.html

Reply:
@nvlsianpu one more time...

Reply:
SI commit emphasizes that it is security weakness (the grade of issue severity is important). 
Please correct - vulnerability -> weakness.